### PR TITLE
Add Meirin

### PR DESCRIPTION
**Pull request by meevans1**

### CODE REVIEW

File: _data/people/meevans1.yml [hunk 1]
```diff
@@ -0,0 +1,68 @@
+name: <Meirin Oan Evans>
```
Comment (henryiii) [inline _data/people/meevans1.yml]: No `<>` brackets, please. :)

File: _data/people/meevans1.yml [hunk 2]
```diff
@@ -0,0 +1,68 @@
...
+    meeting: 4th Inter-experiment Machine Learning Workshop
+    meetingurl: https://indico.cern.ch/event/852553/
+    location: Online
+  - title: Introduction to Machine Learning
+    date: 2020-10-12
+    url: https://indico.ph.liv.ac.uk/event/103/contributions/1221/
+    meeting: STFC School on Data Intensive Science 2020
+    meetingurl: https://indico.ph.liv.ac.uk/event/103
+    location: Online
+  - title: Connecting teaching to research: ATLAS Open Data in Sussex courses
```
Comment (henryiii) [inline _data/people/meevans1.yml]:
```suggestion
  - title: "Connecting teaching to research: ATLAS Open Data in Sussex courses"
```

Colon followed by space is not allowed in YAML. It makes a nested dict instead. (or something like that, it really just crashes in practice)

File: _data/people/meevans1.yml [hunk 3]
```diff
@@ -0,0 +1,68 @@
...
+    meeting: ICHEP 2020
+    meetingurl: https://indico.cern.ch/event/868940
+    location: Online
+  - title: LHC Open Data for the world to see
+    date: 2020-05-29
+    url: https://indico.cern.ch/event/856696/contributions/3855377/
+    meeting: LHCP 2020
+    meetingurl: https://indico.cern.ch/event/856696
+    location: Online
+  - title: ATLAS Open Data: Data visualisation and educational physics analysis to re-discover the Higgs
```
Comment (henryiii) [inline _data/people/meevans1.yml]:
```suggestion
  - title: "ATLAS Open Data: Data visualisation and educational physics analysis to re-discover the Higgs"
```

File: _data/people/meevans1.yml (hunk 1 above)
Comment (meevans1) [reply _data/people/meevans1.yml]: Resolved in commit https://github.com/iris-hep/iris-hep.github.io-source/pull/534/commits/420ebd062d598e9fc399a5e99d653b2da082e198

**Comment by henryiii** [comment]
From the logs:

```
meevans1 is not listed in a university and hidden is not True 
```

**Comment by meevans1** [comment]
Thanks @henryiii for sorting this out. Is there anything I need to do?

**Pull request merged**